Make error in capture.cpp Ubuntu 20.04

The following error occurs when compiling the actual release under Ubuntu 20.04. Can this be corrected? I use a powerful virtual machine to create startrail, keogram and video. Many thanks

pi@allsky:~/allsky01$ make -C src all
make: Entering directory '/home/pi/allsky01/src'
2021-11-03 09:49:13 Building capture program...
capture.cpp: In function ‘void* SaveImgThd(void*)’:
capture.cpp:298:18: error: ‘cvGetTickCount’ was not declared in this scope
  298 |             st = cvGetTickCount();
      |                  ^~~~~~~~~~~~~~
make: *** [Makefile:122: capture] Error 1
make: Leaving directory '/home/pi/allsky01/src'

--- a/src/capture.cpp
+++ b/src/capture.cpp
@@ -295,7 +295,7 @@ void *SaveImgThd(void *para)
 	    // so set "cmd" before imwrite().
 	    // The temperature must be a 2-digit number with an optional "-" sign.
             sprintf(cmd, "%s %s '%s' '%2.0f' %ld &", s, dayOrNight.c_str(), fileName, (float) actualTemp/10, current_exposure_us);
-            st = cvGetTickCount();
+            st = cv::getTickCount();
             try
             {
                 result = imwrite(fileName, pRgb, compression_parameters);
@@ -304,7 +304,7 @@ void *SaveImgThd(void *para)
             {
                 printf("*** ERROR: Exception saving image: %s\n", ex.what());
             }
-            et = cvGetTickCount();
+            et = cv::getTickCount();
 
             if (result)
                 system(cmd);

